doc/install.sh

```diff
@@ -64,19 +64,19 @@ fi
 #
 THE_ZDOTDIR="${ZDOTDIR:-$HOME}"
 RCUPDATE=1
-if egrep '(zinit|zplugin)\.zsh' "$THE_ZDOTDIR/.zshrc" >/dev/null 2>&1; then
+if grep -E '(zinit|zplugin)\.zsh' "$THE_ZDOTDIR/.zshrc" >/dev/null 2>&1; then
     echo "[34m▓▒░[0m .zshrc already contains \`zinit …' commands – not making changes."
     RCUPDATE=0
 fi
 
 if [ $RCUPDATE -eq 1 ]; then
     echo "[34m▓▒░[0m Updating $THE_ZDOTDIR/.zshrc (10 lines of code, at the bottom)"
-    ZINIT_HOME="$(echo $ZINIT_HOME | sed "s|$HOME|\$HOME|")"
+    ZINIT_HOME="$(echo "$ZINIT_HOME" | sed "s|$HOME|\$HOME|")"
```

Comment:
This can be done entirely in Zsh

Reply:
Yeah the entire script could in zsh. What's your point?

Reply:
Ah, I missed that it was in `/bin/sh`.  My bad.